roles/kubernetes/master/tasks/kubeadm-secondary-experimental.yml

```diff
@@ -49,7 +49,10 @@
     msg: "{{ kubeadm_already_run.stat.exists }}"
 
 - name: Joining control plane node to the cluster.
-  command: >-
+  shell: >-
+    if [ -f /etc/kubernetes/manifests/kube-apiserver.yaml ]; then
+    {{ bin_dir }}/kubeadm reset -f --cert-dir {{ kube_cert_dir }};
+    fi &&
```

Comment:
If `/etc/kubernetes/manifests/kube-apiserver.yaml` does not exists, then this will result in the command starting with &&

I guess you'd need to put the && into the command line, inside the if statement?

Reply:
This bash works just fine. Moving && inside the if will break it.
```
$ if [ 1 -lt 0 ]; then  echo less; fi && echo always
always
$ if [ 0 -lt 1 ]; then  echo less; fi && echo always
less
always
```